infra/docker/python_base/Dockerfile

```diff
@@ -12,15 +12,17 @@ RUN pip install virtualenv
 WORKDIR /opt/filter_lib
 RUN python3 -m venv env
 RUN . env/bin/activate && pip install -r requirements.txt \
-    && pip install -r requirements-dev.txt && isort src --check \
-    && black src --check && mypy src && pylint src \
+    && pip install -r requirements-dev.txt \  
+    # && isort src --check \
+    # && black src --check && mypy src && pylint src \
```

Comment:
I believe, these lines were commented only for development purposes

Reply:
removed